core/input/input_event.cpp

```diff
@@ -1758,10 +1785,6 @@ void InputEventShortcut::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "shortcut", PROPERTY_HINT_RESOURCE_TYPE, "Shortcut"), "set_shortcut", "get_shortcut");
 }
 
-bool InputEventShortcut::is_pressed() const {
-	return true;
-}
-
```

Comment:
Is there a reason for the `InputEventShortcut` to be in the `pressed` state?

Reply:
I don't know for sure, but I guess it's because they are not actual input events and a conceptual decision had to be made about whether to consider such a fake event type as pressed or not, with the former making the most sense.

Reply:
With the refactor, its state should be `NONE` (or `DEFAULT` depending on what we settle on) which would change `is_pressed()` result to `false`.
I'll go ahead with that update, and can always revert if that breaks some underlying assumption.